scripts/generate_bibliography.rb

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require "bundler/setup"
+require "yaml"
+require "relaton"
+
+class GenerateBibliography
```

Comment:
This code should be integrated into geolexica-server. It should not be at the level of the site deployment.

Reply:
@ronaldtse currently, the `Makefile` and `scripts` for generating metadata are in this repo so that is why I added this script here as well. 

If we add the script in `geolexica-server` then we will also have move the `Makefile` there and also extract the path for `bibliography.yaml` because that will be different based on the repo. e.g for this repo the path for `bibliography.yaml` is `isotc201-glossary/bibliography.yaml` for `isotc211` the path will be `isotc211-glossary/bibliography.yaml`.

Reply:
From what I understand, I will be doing the following 

- moving the `generate_bibliography.rb` script to `bibliography_generator.rb` in `geolexica-server`
- add the path of `bibliography.yaml` file in the config of each dataset i.e `isotc204.geolexica.org/_config.yaml`, `isotc211.geolexica.org/_config.yaml`
- reverting changes in current repo

Reply:
I've opened a PR [here](https://github.com/geolexica/geolexica-server/pull/194) for `geolexica-server`.
@ronaldtse Can you review it and let me know if I'm going in the right direction?

Reply:
Yes the steps are correct. Replied in the PR. Thanks!

Reply:
I actually wonder if we should just have a new gem that works with the "concepts dataset". e.g. enumerate the concepts, load each concept into a class, read/write concepts of the dataset, validate the dataset, etc.

Reply:
Ping @HassanAkbar perhaps the Glossarist gem should do the compilation/prep of a concepts dataset?